From 18da42b5ac73e952a59f7d3ad79cd826bb7645d3 Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Wed, 16 Oct 2019 16:22:52 +0800
Subject: [PATCH] arm64: rockchip_defconfig: add video codec configs

add configs:
CONFIG_ROCKCHIP_MPP_RKVDEC
CONFIG_ROCKCHIP_MPP_VDPU1
CONFIG_ROCKCHIP_MPP_VDPU2
CONFIG_ROCKCHIP_MPP_VEPU1
CONFIG_ROCKCHIP_MPP_VEPU2

Change-Id: I384690294c45c88b2a9125827cb796b15f9cdba0
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index e19fb69874f2..fedf7d6cab0b 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -585,10 +585,13 @@ CONFIG_BACKLIGHT_CLASS_DEVICE=y
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_ROCKCHIP_RGA2=y
-CONFIG_RK_VCODEC=y
 CONFIG_IEP=y
 CONFIG_ROCKCHIP_MPP_SERVICE=y
-CONFIG_ROCKCHIP_MPP_DEVICE=y
+CONFIG_ROCKCHIP_MPP_RKVDEC=y
+CONFIG_ROCKCHIP_MPP_VDPU1=y
+CONFIG_ROCKCHIP_MPP_VDPU2=y
+CONFIG_ROCKCHIP_MPP_VEPU1=y
+CONFIG_ROCKCHIP_MPP_VEPU2=y
 CONFIG_LOGO=y
 # CONFIG_LOGO_LINUX_MONO is not set
 # CONFIG_LOGO_LINUX_VGA16 is not set
-- 
2.35.3

